mesh/ios/AVPlayerPlugin/AVPlayerMesh/AppDelegate.swift

```diff
@@ -1,13 +1,22 @@
 //
 //  AppDelegate.swift
 //  AVPlayerMesh
+//
+//  Created by Mehala, Damien on 14/01/2022.
+//
```

Comment:
I think we wanted to removes these kind of lines?

Reply:
done